src/lightning_habana/__about__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "1.3.0"
+__version__ = "1.4.0-dev0"
```

Comment:
```suggestion
__version__ = "1.3.1"
```
Maybe release it as quick bugfix?

Reply:
Are there any fabric specific models that you are aware of ?

Reply:
Will merge this now. If its critical, we'll make a release out of this.

Reply:
Yes but release it as 1.3.1, not 1.4.0